dev/ledger.md

```diff
@@ -583,6 +621,49 @@ and contains the following fields:
   in the reference implementation, one of these two fields will be zero
   in that case.
 
+- If this is an `ApplicationCall` transaction, the `EvalDelta` associated with the successful execution of the corresponding application's `ApprovalProgram` or `ClearStateProgram`. The `EvalDelta`, encoded as msgpack field `dt`, contains the following fields:
+  - A `GlobalDelta`, encoding changes to the global state of the called application, encoded as msgpack field `gd`.
+    - `gd` is a [`StateDelta`][State Deltas].
+  - Zero or more `LocalDeltas`, encoding changes to some local states associated with the called applicaiton, encoded as msgpack field `ld`.
+    - `ld` maps an "account offset" to a [`StateDelta`][State Deltas]. Account offset 0 is the transaction's sender. Account offsets 1 and greater refer to the account specified at that index minus one in the transaction's `Accounts` slice.
+
+TEAL Key/Value Stores
+---------------------
+A TEAL Key/Value Store, or TKV, is an associative array mapping keys of type byte-array to values of type byte-array or 64-bit unsigned integer.
+
+The values in a TKV are represented by the `TealValue` struct, which is composed of three fields:
+
+- `Type`, encoded as msgpack field `tt`. This field may take on one of two values:
+  - `TealBytesType` (value = `1`), indicating that the value can be found in the `Bytes` field of this struct.
+  - `TealUintType` (value = `2`), indicating that the value can be found in the `Uint` field of this struct.
+- `Bytes`, encoded as msgpack field `tb`, representing a byte slice value.
+- `Uint`, encoded as msgpack field `ui`, representing an unsigned 64-bit integer value.
+
+The keys in a TKV are encoded directly as bytes.
+
+State Deltas
+------------
+
+A state delta represents an update to a [TEAL Key/Value Store (TKV)][TEAL Key/Value Stores]. It is represented as an associative array mapping a byte-array key to a single value delta.
+
+A value delta is composed of three fields:
+
+- `Action`, encoded as msgpack field `at`, which specifies how the value for this key has changed. It has three possible values:
+  - `SetUintAction` (value = `1`), indicating that the value for this key should be set to the value delta's `Uint` field.
+  - `SetBytesAction` (value = `2`), indicating that the value for this key should be set to the value delta's `Bytes` field.
+  - `DeleteAction` (value = `3`), indicating that the value for this key should be deleted.
+- `Bytes`, encoded as msgpack field `bs`, which specifies a byte slice value to set.
+- `Uint`, encoded as msgpack field `ui`, which specifies a 64-bit unsigned integer value to set.
+
+State Schemas
+-------------
```

Comment:
It might make sense for this to be a subsection

Reply:
done